internal/load/common.go

```diff
@@ -63,6 +63,9 @@ func NewKubeAccess() (*KubeAccess, error) {
 		return nil, err
 	}
 
+	restConfig.QPS = 500
+	restConfig.Burst = 500
```

Comment:
I am not familiar with those settings. Could you add a comment to explain the reason? Thanks.

Reply:
Yeah, I basically wanted to produce many builds at once to put the build controller under pressure. But with the default QPS (5) this was not very efficient. The 500, I arbitrarily selected.